javascript/javascript3/week2/index.js

```diff
@@ -0,0 +1,94 @@
+//Movies exercise ex------------ (1)
+const movieUrl = "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+async function moviefunction(){
+try{
+const movieResponse = await fetch(movieUrl);
+const movieData = await movieResponse.json();
+////Create an array of bad movies
+const badMovie = movieData.filter((movie)=>movie.rating<=5);
+const badMovieTitle = badMovie.map((movie)=>movie.title);
+console.log("badmovies : ",badMovieTitle);
+//Creat an array of bad movies since year 2000
+const badMovie2000 = badMovie.filter((movie)=>movie.year>=2000);
+const badMovie2000Title = badMovie2000.map((movie)=>movie.title);
+console.log("badmovis since 2000 : ",badMovie2000Title);
+}
+catch(error){
+console.log(error);
+}
+}
+moviefunction();
+
+//Promise that resolves after set time ex-----------(2)
+function result_delay(resolveAfter) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(console.log("I´m async response"));
```

Comment:
Using the log statement is weird here
just call resolve like this
```js
setTimeout(() => {
      resolve()
```

Reply:
I will change my code as per your feedback ..Thanks again.. :)